diffkemp/simpll/CMakeLists.txt

```diff
@@ -1,16 +1,20 @@
 # Create symlinks to a correct version of FunctionComparator based on LLVM version
-execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
+execute_process(
+  COMMAND ${CMAKE_COMMAND} -E create_symlink
```

Comment:
Please do not reformat lines that are not related to this PR, it makes `git blame` useless (this applies to almost the entire file).

Reply:
My mistake, my vim autocmd was at fault and I realised as much before, which is made me run clang-format but then I forgot about the CMakeLists